.github/PULL_REQUEST_TEMPLATE.md

```
@@ -18,13 +23,15 @@
 <!-- 이 PR에서 테스트한 내용을 설명해주세요. -->
 - [ ] 목표한 구현 정상 동작 확인
+- [ ] 엣지 케이스 테스트 완료
+- [ ] 기존 기능 영향 없음
 
 ## 🎨 스크린샷
 <!-- UI 변경사항이 있는 경우 스크린샷을 첨부해주세요. -->
 <!-- img src "이부분에 gif파일 넣어주세요" -->
 |기능|스크린샷|
 |:--:|:--:|
-|GIF|<img src = "" width ="250">|
+|GIF|<img src="https://..." width="250" />|
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)